configs/roles/inventory.json

```
@@ -0,0 +1,16 @@
+{
+  "roles": [
+    {
+      "name": "Inventory Administrator",
+      "description": "An Inventory Administrator role that grants read and write permissions to all Inventory resources.",
+      "system": true,
+      "platform_default": true,
+      "version": 1,
```

Due to an idiosyncrasy in the way our seeding works, this needs to be started at `"version": 2`.
```suggestion
      "version": 2,
```

---

Thanks! Done! :+1: